src/test/java/org/broadinstitute/hellbender/tools/exome/ExomeToolsTestUtils.java

```diff
@@ -34,42 +34,57 @@ protected static File getTestDataDir(){
      */
     protected final static SAMSequenceDictionary REFERENCE_DICTIONARY = SAMSequenceDictionaryExtractor.extractDictionary(REFERENCE_FILE);
 
+    protected static Comparator<SimpleInterval> createIntervalComparator() {
```

Comment:
So are you suggesting to create a "standard" comparator as a constant in SimpleInterval? Say
SimpleInterval.LEXICALLY_SORTED_CONTIG_COMPARATOR? A better suggestion for the name?